ios/RNZoomBridge.m

```diff
@@ -256,12 +256,16 @@ - (void)onMeetingError:(MobileRTCMeetError)errorCode message:(NSString *)message
     return;
   }
 
+   if (errorCode != MobileRTCMeetError_Success || errorCode != 0) {
```

Comment:
Similar change here. And since we're clearing the promise let's resolve it before we do

So I think we should change this to:

```
if ((errorCode != MobileRTCMeetError_Success) && (errorCode != MobileRTCMeetError_MeetingOver)) {
  meetingPromiseReject(
    @"ERR_ZOOM_MEETING",
    [NSString stringWithFormat:@"Error: %d, internalErrorCode=%@", errorCode, message],
    [NSError errorWithDomain:@"us.zoom.sdk" code:errorCode userInfo:nil]
  );
} else {
    meetingPromiseResolve(@"[onMeetingError] Connected to zoom meeting");
}
```